yarn.lock

```diff
@@ -7993,6 +7993,16 @@ jss@10.6.0, jss@^10.5.1:
     is-in-browser "^1.1.3"
     tiny-warning "^1.0.2"
 
+jsverify@^0.8.4:
```

Comment:
I don't see any `package.json` changes; how did we get a change to `yarn.lock`?

Reply:
We added jsverify to do algorithmic testing. It was too expensive for CI, so we've dropped it. Thanks for catching this change I missed.